tests/functional/s/super/super_checks.py

```diff
@@ -123,3 +123,34 @@ class SuperWithSelfClass(object):
     """self.__class__ may lead to recursion loop in derived classes"""
     def __init__(self):
         super(self.__class__, self).__init__() # [bad-super-call]
+
+
+# Reported in https://github.com/PyCQA/pylint/issues/2903
+class Parent:
+    def method(self):
+        print()
+
+
+class Child(Parent):
+    def method(self):
+        print("Child")
+        super().method()
+
+class Niece(Parent):
+    def method(self):
+        print("Niece")
+        super().method()
+
+class GrandChild(Child):
+    def method(self):
+        print("Grandchild")
+        super(GrandChild, self).method()
+        super(Child, self).method()
+        super(Niece, self).method()  # [bad-super-call]
+
+
+# Reported in https://github.com/PyCQA/pylint/issues/4922
+class AlabamaCousin(Child, Niece):
```

Comment:
Well, that's very funny 😄 ``CarlosTwoOfSpain`` works too if we want to keep the goodwill of alabamians 😛